templates/admin/adherent/edit.html.twig

```diff
@@ -1,6 +1,13 @@
 {% extends 'SonataAdminBundle:CRUD:base_edit.html.twig' %}
 
 {% block actions %}
+    <script>
+        $(function(){
+            $('.ban-link').click(function(){
+                $('#ban').submit();
```

Comment:
why do you not use `<input type=submit>` button?

Reply:
I prefer to keep the `<a href>` to have the same design of links.
If I add the `input` I need to add the click event on this input and IMHO it's no necessary. I prefer to submit directly the form. WDYT ?